src/main/steps/common/common.content.ts

```diff
@@ -309,22 +310,28 @@ export const generatePageContent = ({
 };
 
 const getServiceName = (
-  reqData: CommonContentAdditionalData | undefined,
-  translations: typeof en | typeof cy
+  translations: typeof en | typeof cy,
+  userCase: Partial<CaseWithId> | undefined,
+  url: string | undefined
 ): string => {
-  const url = reqData?.path;
-  const isCommonServiceName = url?.includes(DASHBOARD_URL) || ANONYMOUS_URLS.some(_url => _url.includes(url));
-  const isC100 = url?.startsWith(C100_URL) || reqData?.session?.userCase?.caseTypeOfApplication === C100_CASE_TYPE.C100;
-  let serviceName;
+  let serviceName = translations.commonServiceName;
 
-  if (isCommonServiceName) {
-    serviceName = translations.commonServiceName;
-  } else {
-    if (isC100) {
-      serviceName = translations.c100ServiceName;
-    } else {
-      serviceName = translations.fl401ServiceName;
-    }
+  if (!url ||
```

Comment:
Can you create a array called common pages and put all these pages in the array and check against it